src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -526,45 +695,61 @@ private Vector2[] GetLeastIntersectingPath(PatchRegion start, PatchRegion end)
         var right = startRect.End.X > endRect.End.X ? startRect : endRect;
 
         // 3-segment line, Z shape
-        var middlePoint = new Vector2(left.End.X + right.Position.X, upper.End.Y + lower.Position.Y) / 2.0f;
+        for (int i = -3; i <= 3; ++i)
+        {
+            var middlePoint =
+                new Vector2(left.End.X + right.Position.X + i * 25, upper.End.Y + lower.Position.Y + i * 25) / 2.0f;
```

Comment:
Also I know that the old code did this, but could you update any `/ 2.0f` pieces of code to be `* 0.5f` where they are safe to change? Floating point multiplies are faster to calculate than divides so it should be preferred whenever a divide is not absolutely required.